ci-operator/config/openshift/release/openshift-release-master__nightly-4.15.yaml

```diff
@@ -587,6 +592,18 @@ tests:
     cluster_profile: gcp
     workflow: openshift-e2e-gcp-xpn
   timeout: 6h0m0s
+- as: e2e-aws-ovn-upgrade-rollback-oldest-supported
+  interval: 48h
+  steps:
+    cluster_profile: aws-2
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:oldest-supported-z-rollback
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:latest
```

Comment:
Does this say install release:oldest-supported-z-rollback and upgrade to latest?

I'm thinking we want to install latest and upgrade to oldest-supported-z-rollback but might be misreading it

Reply:
I am simply following the pattern of the other rollback job.  While in this case, I suspect that this would have the same result:
```
OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:latest
OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:initial
```
However, the end game here will be to update the `oldest-supported-z-rollback` definition to point to something other than the "latest accepted release".  At this time we can't point back to an EC release, but I think that that would be the idea here.